{{ cookiecutter.package_name }}/.github/workflows/ci_tests.yml

```diff
@@ -0,0 +1,79 @@
+# GitHub Actions workflow for testing and continuous integration.
+#
+# This file performs testing using tox and tox.ini to define and configure the test environments.
+
+name: CI Tests
+
+on: [push, pull_request]
```

Comment:
Should we also include @saimn 's patch in astropy/astropy#11054 so Actions won't also run on forks when PR is opened?

Reply:
yes, this keeps the template in-line with what astropy is doing so definitely worth doing.